xpano/log/logger.cc

```diff
@@ -71,6 +71,9 @@ void Logger::RedirectSpdlogOutput(
 
   if (app_data_path) {
     auto log_path = *app_data_path / kLogFilename;
+    // set log_file_path_ to *app_data_path to use in bugreport_pane
+    log_file_path_ = (*app_data_path).string();
```

Comment:
Please write `app_data_path->string()`

Please also remove the comment, or just leave "// Used in bug report window"